operators/multiclusterobservability/manifests/base/grafana/dash-acm-virtual-machines-inventory.yaml

```diff
@@ -0,0 +1,322 @@
+apiVersion: v1
+data:
+  acm-openshift-virtual-machines-inventory.json: |-
+    {
+      "annotations": {
+        "list": [
+          {
+            "builtIn": 1,
+            "datasource": "-- Grafana --",
+            "enable": true,
+            "hide": true,
+            "iconColor": "rgba(0, 211, 255, 1)",
+            "name": "Annotations & Alerts",
+            "target": {
+              "limit": 100,
+              "matchAny": false,
+              "tags": [],
+              "type": "dashboard"
+            },
+            "type": "dashboard"
+          }
+        ]
+      },
+      "description": "This dashboard provides a comprehensive list of all virtual machines across clusters, offering detailed insights into each VM’s configuration. Users can easily search and filter VMs based on various attributes, enabling quick access to specific information for effective management and troubleshooting.",
```

Comment:
Might be worth getting @dockerymick to do a review of the copy on these dashboards. I think there's a style guide from the doc writers they would probably like to maintain here as well.

Reply:
```suggestion
      "description": "This dashboard provides a comprehensive list of all virtual machines deployed in a given managed cluster, offering detailed insights into each VM’s configuration. Users can easily search and filter VMs based on their attributes, enabling quick access to specific information for effective management and troubleshooting.",
```